src/metabase/search/postgres/core.clj

```diff
@@ -0,0 +1,81 @@
+(ns metabase.search.postgres.core
+  (:require
+   [honey.sql :as sql]
+   [honey.sql.helpers :as sql.helpers]
+   [metabase.api.common :as api]
+   [metabase.search.config :as search.config]
+   [metabase.search.impl :as search.impl]
+   [metabase.search.postgres.index :as search.index]
+   [metabase.search.postgres.ingestion :as search.ingestion]
+   [toucan2.core :as t2]))
+
+(defn- user-params []
```

Comment:
this looks like it's used only in REPL? in this case I'd leave this in the dev/test code and not confuse this implementaiton.

Reply:
It's also used [here](https://github.com/metabase/metabase/pull/47752/files#diff-68bda3e0092ace30a4eef5614ea9d25a4d463ffe37afe045e1a8bb34900a3225R25) in the application code.

Reply:
oh I see, then why do we need to have this custom context? why cant' we use the same context that `search` function takes?

Reply:
I wanted to capturing the global state in one place, so that all the top-level functions work from the REPL without needing to go through some wrapper in the dev namespace.

You're totally right that it should use the user parameters from `search-ctx` though, I'll clean that up.